9.2/root/usr/share/container-scripts/postgresql/README.md

```diff
@@ -33,6 +33,9 @@ You can also set the following mount points by passing the `-v /host:/container`
 | :----------------------- | ------------------------------------- |
 |  `/var/lib/pgsql/data`   | PostgreSQL database cluster directory |
 
+
+At location `${CONTAINER_SCRIPTS_PATH}/post-init.sh` should be located script which defines all post initialization tasks. By default there is no script and only way how to get script into location is using this image as base image. In this case you should do `COPY path-to-your-script ${CONTAINER_SCRIPTS_PATH}/post-init.sh`.
```

Comment:
@pkajaba could you explain here some use cases why people would want to use a post-init script?
As you mentioned, database migration, loading initial data, etc...